src/main/java/io/reactivex/processors/BehaviorProcessor.java

```diff
@@ -175,10 +175,8 @@ public void onSubscribe(Subscription s) {
 
     @Override
     public void onNext(T t) {
-        if (t == null) {
-            onError(new NullPointerException("onNext called with null. Null values are generally not allowed in 2.x operators and sources."));
```

Comment:
😰😰😰 well this is a little bit breaking change, but since it was always part of RS spec I guess it falls into patch category for RxJava

Reply:
You were not supposed to call onNext with null anyway and there is no reason to take this extra long path to get the end subscriber a `NullPointerException` via `onError`.

Reply:
Yep I understand :) 

Just noted that it's a little bit breaking behavior for existing users, but since RS doesn't allow nulls anyway we can make this change as "patch" rather than wait for 3.x